aksel.nav.no/website/pages/gp/index.tsx

```diff
@@ -61,51 +104,141 @@ const GpPage = (props: PageProps["props"]) => {
 
   return (
     <>
-      {/* TODO: Find out how we want to handle SEO for these pages */}
       <SEO
         title="God praksis"
-        /* description={page?.seo?.meta} */
-        /* image={page?.seo?.image} */
+        description={props.page?.seo?.meta}
+        image={props.page?.seo?.image}
       />
-      <GodPraksisPage {...props} />
+
+      <Page
+        footer={<Footer />}
+        footerPosition="belowFold"
+        className="bg-surface-subtle"
+      >
+        <Header variant="subtle" />
+        <Box
+          paddingBlock="10"
+          as="main"
+          tabIndex={-1}
+          id="hovedinnhold"
+          className="animate-popUpPage focus:outline-none"
+        >
+          <Page.Block width="xl" gutters>
+            <VStack gap="10">
+              <IntroHero title="God praksis">
+                {props.page.intro && (
+                  <SanityBlockContent
+                    isIngress
+                    blocks={props.page.intro}
+                    className="mt-4"
+                  />
+                )}
+                <nav aria-label="Temavelger">
+                  <Stack
+                    className="mt-6"
+                    gap={{ xs: "4", md: "6" }}
+                    wrap
+                    direction={{ xs: "column", md: "row" }}
+                    as="ul"
+                  >
+                    {props.tema
+                      .filter((x) => x.refCount > 0)
+                      .map((tema) => (
+                        <li key={tema.slug}>
+                          <GpHeroCard
+                            articleCount={tema.refCount}
+                            href={`gp/${tema.slug}`}
+                            image={tema.pictogram}
+                          >
+                            {tema.title}
+                          </GpHeroCard>
+                        </li>
+                      ))}
+                  </Stack>
+                </nav>
+              </IntroHero>
+              <Box paddingInline={{ xs: "4", lg: "10" }}>
+                <VStack gap="12">
+                  {props.tema
+                    .filter((x) => x.refCount > 0)
+                    .map((tema) => {
+                      return (
+                        <section
+                          key={tema.slug}
+                          aria-label={`Tema ${tema.title}`}
+                        >
+                          <Heading
+                            level="2"
+                            size="medium"
+                            className={cl("text-aksel-heading", {
+                              "mb-4": !tema.description,
+                              "mb-2 ": !!tema.description,
```

Comment:
```suggestion
                              "mb-2": tema.description,
```
Ekstra mellomrom pluss tror ikke `!!` trengs.